### PR TITLE
RA-1352: Fixed field separation display for names

### PR DESCRIPTION
**Pull request by benledoux**
We have made changes to the uicommons module that will allow all Name IDs to display correctly when a name has been input. Note: This has been checked across all naming convention methods from the Administration menu.

### CODE REVIEW

**Comment by ODORA0** [comment]
Did you have a look at this https://wiki.openmrs.org/display/docs/Pull+Request+Tips?

File: omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js [hunk 1]
```diff
@@ -662,9 +663,14 @@ ConfirmationSectionModel.prototype.select = function() {
...
-                        summaryDiv.append("<p><span class='title'>" + question.title().text() + ": </span>"
+                        if (question.title().text() == "Name") {
+                            summaryDiv.append("<p><span class='title'>" + question.title().text() + ": </span>"
+                            + (question.valueAsTextNoSeparator) + "</p>");
+                        }
+                        else {
+                            summaryDiv.append("<p><span class='title'>" + question.title().text() + ": </span>"
                             + (question.valueAsText && !/^\s*$/.test(question.valueAsText) ? question.valueAsText : "--") + "</p>");
-
+                        }
```
Comment (ODORA0) [inline omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js]: Are these formatting changes necessary?

Comment (benledoux) [reply omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js]: Can you please specify which specific formatting change you are referring to? We altered this code to specifically find the name value and remove the field separators in regards to RA-1352.

**Comment by mozzy11** [comment]
@benledoux  , can you squash your commits into one as advised here 
https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by brandones** [comment]
Closing as stale.

**Pull request closed without merging**